File: 02-java/demo-01/UnitTestingAndTDD/Sample02/src/main/java/sample02/abstractions/IFizzBuzzService.java
package sample02.abstractions;

public interface IFizzBuzzService {
    public final String MultipleOf3 = "Fizz";
    public final String MultipleOf5 = "Buzz";
    public final String MultipleOf3And5 = "FizzBuzz";
    public final String IllegalArgumentExceptionMaxMinorThanMin = "The max parameter must be greater than min parameter.";
    public final String IllegalArgumentExceptionMinMustBePositive = "The min parameter must be positive";

    String multipleOf3(int number);

    String multipleOf5(int number);

    String multipleOf3And5(int number);
}

File: 01-csharp/demo-01/UnitTestingAndTDD/Sample02/Abstractions/IFizzBuzzService.cs
namespace Sample02.Abstractions
{
    public interface IFizzBuzzService
    {
        public const string MultipleOf3Result = "Fizz";
        public const string MultipleOf5Result = "Buzz";
        public const string MultipleOf3And5Result = "FizzBuzz";
        public const string IllegalArgumentExceptionMaxMinorThanMin = "The max parameter must be greater than min parameter.";
        public const string IllegalArgumentExceptionMinMustBePositive = "The min parameter must be positive";

        public string MultipleOf3(int number);

        public string MultipleOf5(int number);

        public string MultipleOf3And5(int number);  
    }
}